ITRStartTask/AppLibrary/ApiService/ImageDownloader/ImagesDownloader.swift

```diff
@@ -10,36 +10,45 @@ import UIKit
 
 class ImagesDownloader: ImagesDownloaderProtocol {
     private let queue = OperationQueue()
-    private var operations: [Int: ImageDownloadOperation] = [:]
+    private var operations: [String: ImageDownloadOperation] = [:]
 
-    weak var delegate: ImagesDownloaderDelegate?
+    func getImage(withUrl urlString: String,
+                  onSuccess: @escaping (UIImage) -> Void,
+                  onFailure: @escaping (Error) -> Void) {
+        let urlHash = self.getHash(forString: urlString)
```

Comment:
!!! Hash should be global (static) function or standard function should be used.

Reply:
fixed